contrib/rpm/README.md

```diff
@@ -0,0 +1,185 @@
+RPM Spec File Notes
+-------------------
+
+The RPM spec file provided here is for Bitcoin-Core 0.12.0 and builds on CentOS
```

Comment:
Is is useful to distribute those RPM spec files via git, considering that they need update/bump *prior* to each release? (Including them in `master` for 0.12.0 makes them somewhat less accessible, compared to having them in the tag `v0.12.0` or a separate place)